swaynag/types.c

```diff
@@ -100,6 +101,10 @@ void swaynag_type_merge(struct swaynag_type *dest, struct swaynag_type *src) {
 		dest->anchors = src->anchors;
 	}
 
+	if (src->layer > 0) {
```

Comment:
`ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND == 0` so we can't use this as a guard value.

Reply:
> `ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND == 0` so we can't use this as a guard value.

Thanks for spotting this, I've updated it to use a `int32_t` with an initial value of `-1`.

We could alternatively remove support for the `background` setting, as I'm unaware of the use case. From my testing the nag bar appears behind the desktop background, but if a smaller bg is centred on the page it does show and looks similar to using a value of `bottom`.